lib/index.js

```diff
@@ -215,13 +203,69 @@ function lowerCaseObjectKeys(obj) {
   return newObject;
 }
 
+function validateDate(date, key) {
```

Comment:
Please move all helper functions to the bottom of the file (below public API functions). This helps with readability/maintenance.

Reply:
Addressed here: https://github.com/digitalbazaar/http-signature-header/pull/17/commits/909d594a20d06b0963b1ea605baad00dd8c27ab4